source/manual/debian-packaging.html.md

```diff
@@ -187,16 +188,16 @@ deb file to aptly.
 by checking the diff. You can find the previous snapshot by running `sudo aptly snapshot list`
 
     ```
-      $ sudo aptly snapshot diff rbenv-ruby-20181023 rbenv-ruby-20190212
+      $ sudo aptly snapshot diff rbenv-ruby-20181023 $snapshot
         Arch   | Package          | Version in A  | Version in B
       + amd64  | rbenv-ruby-2.6.1 | -             | 1
 
     ```
 6. Publish the new snapshot, you will be prompted to enter the passphrase for our
-APT account which is in [govuk-secrets](https://github.com/alphagov/govuk-secrets/tree/master/pass) `PASSWORD_STORE_DIR=~/govuk/govuk-secrets/pass/2ndline pass apt`
+APT account which is in [govuk-secrets](https://github.com/alphagov/govuk-secrets/tree/master/pass) `./edit.sh 2ndline apt/passphrase`
```

Comment:
Hmm... I don't get problems with the `.gpg-id` files when using `pass` to get the secret, only if I use `pass` directly for editing or deleting.

Reply:
Oh, it's been a while since I tried that, maybe it is fine